code/modules/power/cables/cable.dm

```diff
@@ -140,9 +140,7 @@ By design, d1 is the smallest direction and d2 is the highest
 	deconstruct()
 
 /obj/structure/cable/proc/cable_color(colorC)
-	if(!colorC)
-		color = COLOR_RED
```

Comment:
Instead of removing these safeties, can you introduce a proper color for white, like `#FEFEFE`? `#FFFFFF` would be preferable, but byond just nulls that out. Maybe specify the define name as `COLOR_CABLE_WHITE`